editoast/src/views/infra/auto_fixes/mod.rs

```diff
@@ -402,11 +396,15 @@ mod tests {
         let app = create_test_service().await;
         let mut small_infra = small_infra(db_pool()).await;
         let small_infra_id = small_infra.id();
+        let conn = &mut db_pool().get().await.unwrap();
         force_refresh(&mut small_infra).await;
 
         // Check the only initial issues are "overlapping_speed_sections" warnings
-        let infra_errors_before_all: PaginatedResponse<crate::views::infra::errors::InfraError> =
-            read_body_json(call_service(&app, errors_request(small_infra_id)).await).await;
+        let infra_errors_before_all = small_infra
+            .model
+            .get_paginated_errors(conn, 1, 25, &QueryParams::default())
```

Comment:
`Deref` is implemented on `TextFixture`.
```suggestion
        let infra_errors_before_all = small_infra
            .get_paginated_errors(conn, 1, 25, &QueryParams::default())
```

There are a lot of other occurrences in the current PR.

Reply:
Done.